CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+## Contributing
+
+```sh
+me$ git clone $repo_url
+cd into_folder
+yarn
```

Comment:
could you change by npm? everyone knows npm and people that know yarn already know that it's a npm replacement